movies/2013/frozen.json

```diff
@@ -0,0 +1,32 @@
+{
+    "name": "Frozen",
+    "year": 2013,
+    "runtime": 102,
+    "categories": [
+      "animation",
+      "adventure",
+      "comedy",
+      "family",
+      "fantasy",
+      "musical"
+    ],
+    "release-date": "2013-11-27",
```

Comment:
this shouldn't be here, please follow the example model [https://github.com/mikeleguedes/json-movie-list/blob/master/readme.markdown](url)